### PR TITLE
Check input value before set it

### LINKED ISSUE #9017: IME does not work properly on input form
## Test plan

1. Enable IME (Japanese/Chinese/Korean/...) on your system
2. Open about:autofill
3. Select 'Add address'
4. Hit some keys
5. Make sure you are seeing the pulldown
6. Hit <kbd>Tab</kbd>
7. Make sure a conversion candidate is selected
8. Hit <kbd>Enter</kbd>
9. Make sure the candidate is decided

----

**Describe the issue you encountered:** IME does not work properly on input form. You cannot select a conversion candidate. 

- Platform (Win7, 8, 10? macOS? Linux distro?): macOS

- Brave Version (revision SHA): https://github.com/brave/browser-laptop/compare/v0.15.303dev...v0.15.304dev

- Steps to reproduce:
    1. Enable Japanese lang input system
    2. Open about:autofill
    3. Select 'Add address'
    4. Hit some keys
- Actual result:

   autofill
  The form is not properly styled as I tried that by reverting it to https://github.com/brave/browser-laptop/blob/c719806d2265434b7e74fce0ef9a7bd061ddcc64/js/components/autofillCreditCardPanel.js to make sure CommonForm is not causing the issue.

  You could confirm the issue on Bookmark Hanger, which appears by clicking the star icon.
  Yet the issue cannot be reproduced on about:preferences#sync.

  autofill3

- Expected result: on 0.15.2

  autofill2

- Is this an issue in the currently released version?
  No.

- Extra QA steps:
    1.
    2.
    3.

- Any related issues: #

### PR DESCRIPTION
**Pull request by darkdh**
## Test Plan:
1. Open editBookmark/autofill address/autofill credit card dialog
2. Make sure IME can input appropriately on text input value
3. Open about:preferences#sync
4. Make sure IME can input appropriately on text input value

## Description

fix #9017

Auditors: @bsclifton, @bbondy

Submitter Checklist:
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [x] New files have MPL2 license header

### CODE REVIEW

File: app/renderer/components/autofill/autofillAddressPanel.js [hunk 1]
```diff
@@ -138,55 +173,102 @@ class AutofillAddressPanel extends ImmutableComponent {
...
-                value={this.props.currentDetail.get('name') || ''}
                 ref={(nameOnAddress) => { this.nameOnAddress = nameOnAddress }} />
               <div className={css(commonFormStyles.input__marginRow)}>
-                <CommonFormTextbox
+                <input className={css(
+                  commonStyles.formControl,
+                  commonStyles.textbox,
+                  commonStyles.textbox__outlineable,
+                  commonStyles.isCommonForm,
+                  commonFormStyles.input__box
```
Comment (luixxiul) [inline app/renderer/components/autofill/autofillAddressPanel.js]: I would make these into a const to avoid repetitive input forms below.

Comment (darkdh) [inline app/renderer/components/autofill/autofillAddressPanel.js]: fixed in 4652c1f

File: app/renderer/components/bookmarks/addEditBookmarkHanger.js [hunk 2]
```diff
@@ -97,12 +97,16 @@ class AddEditBookmarkHanger extends ImmutableComponent {
       this.updateFolders(nextProps)
     }
   }
+  componentDidUpdate () {
+    console.log(this.displayBookmarkName)
```
Comment (bsclifton) [inline app/renderer/components/bookmarks/addEditBookmarkHanger.js]: whoops- we'll want to take this out

Comment (bsclifton) [inline app/renderer/components/bookmarks/addEditBookmarkHanger.js]: I pulled this out in a separate commit and rebased/squashed it in 😄

**Pull request merged**